image/Dockerfile.engine

```diff
@@ -78,8 +78,24 @@ RUN . /binaries-commits && \
 
 # Final docker image
 FROM scratch
+ARG VERSION
+ARG GITCOMMIT
+ARG BUILDTIME
+ARG PLATFORM
 COPY --from=dockerd-builder /sbin/dockerd /sbin/
 COPY --from=proxy-builder /sbin/docker-proxy /sbin/
 COPY --from=init-builder /sbin/docker-init /sbin/
 COPY --from=runc-builder /usr/local/sbin/runc /sbin/
+
+LABEL \
+    org.opencontainers.image.authors="Docker Inc." \
+    org.opencontainers.image.created="${BUILDTIME}" \
+    org.opencontainers.image.documentation="https://docs.docker.com/" \
+    org.opencontainers.image.licenses="Apache-2.0" \
+    org.opencontainers.image.revision="${GITCOMMIT}" \
+    org.opencontainers.image.url="https://www.docker.com/products/docker-engine" \
+    org.opencontainers.image.vendor="Docker Inc." \
+    org.opencontainers.image.version="${VERSION}" \
+    distribution_based_engine="{\"platform\":\"${PLATFORM}\",\"containerd_min_version\":\"1.2.0-beta.1\",\"runtime\":\"host_install\"}"
```

Comment:
Oops, I just pushed an update to my branch, but it was after you merged.

can you prefix this label with "com.docker."

Reply:
Added in https://github.com/docker/docker-ce-packaging/pull/223/commits/4dae5046dac7f1467a611c20befd5cb2eb2aa7a3